priam/src/main/java/com/netflix/priam/restore/AbstractRestore.java

```diff
@@ -271,7 +271,10 @@ public void restore(Date startTime, Date endTime) throws Exception {
             instanceState.setRestoreStatus(Status.FINISHED);
 
             //Start cassandra if restore is successful.
-            cassProcess.start(true);
+            if (!config.doesCassandraStartManually())

```

Comment:
Why is this change in this PR? Is it related or clearing backlog

Reply:
This was one of the misses we had w.r.t. restore. There is already a Jira for this by @jolynch